source/tests/SGM/SGMEnergyTrajectoryTestView.cpp

```diff
@@ -366,7 +367,8 @@ void SGMEnergyTrajectoryTestView::setTrajectoryPlotData(SGMGratingSupport::Grati
 		                               -1.53,
 		                               1000,
 		                               -5598,
-		                               25.07);
+		                               25.07,
+		                               SGMUndulatorSupport::FirstHarmonic);
```

Comment:
I know this is just in your tests code.  But is this second one also supposed to be the first harmonic or should it be the third?